launcher/ui/themes/SystemTheme.cpp

```diff
@@ -54,8 +54,9 @@ SystemTheme::SystemTheme(const QString& styleName, const QPalette& defaultPalett
         m_colorPalette = defaultPalette;
     } else {
         auto style = QStyleFactory::create(styleName);
-        m_colorPalette = style->standardPalette();
-        delete style;
+        m_colorPalette = style != nullptr ? style->standardPalette() : defaultPalette;
+        if (style != nullptr)
+            delete style;
```

Comment:
delete nullptr should be a no-op, right?
source: https://github.com/PrismLauncher/PrismLauncher/pull/3614#discussion_r2038450773

Reply:
.. you are correct. which would explain why it didn't crash before the second commit, I was confused on that